### DISCUSSION TITLE
Simulation "tau" size

### DISCUSSION THREAD

**Discussion by Mario-de-Santi** [upb-lea/gym-electric-motor#248] [Q&A]
Hello again, I would like to know if it is possible to perform simulations with a "tau" larger than the default value of 1e-4. In this case, I would like to simulate with 1e-3, for example. I have already tried to simulate, but it turns out that during the for loop, it considers that the simulation has already ended and activates the env.reset. Is the maximum value I can simulate with 1e-4, or can I use larger simulation steps?

**Comment by max-schenke** [comment]
It should easily be possible to allow other step sizes by specifying e.g. "tau=1e-3" within the gem.make(...) command.

If this is already done correctly, it might be a problem with the controller you employ. 
Possible solutions could be then:
a) The "correct" approach: setting your controller to be less aggressive (when using our default PI controllers this can be done by selecting a larger value for the parameter "a".)
b) The "quick and dirty" approach: deactivate limits, which would allow limit violations without requiring a reset. However, this is unrealistic.

**Reply by Mario-de-Santi** [reply]
Thank you for your attention. I will try to update my controller and make it less aggressive.
